python/src/com/jetbrains/python/console/PyConsoleCopyHandler.kt

```diff
@@ -66,7 +66,9 @@ class PyConsoleCopyHandler(val originalHandler: EditorActionHandler) : EditorAct
       val rangeEnd = Math.min(document.getLineEndOffset(i), end)
       if (rangeStart < rangeEnd) {
         sb.append(document.getText(TextRange(rangeStart, rangeEnd)))
-        sb.append("\n")
+        if (rangeEnd < end) {
+          sb.append("\n")
```

Comment:
If it is an extra \n why not just to trim it and then add \n? I don't quite understand this check.